README.md

```diff
@@ -22,17 +22,15 @@ This is part of the Zulip open source project; see the
 [contributing guide](https://zulip.readthedocs.io/en/latest/overview/contributing.html)
 and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/version-control.html).
 
-1. Fork and clone the Git repo:
+1. Fork and clone the Git repo, and set upstream to zulip/python-zulip-api:
    `git clone https://github.com/<your_username>/python-zulip-api.git`
-
-2. Make sure you have [pip](https://pip.pypa.io/en/stable/installing/)
-   and [virtualenv](https://virtualenv.pypa.io/en/stable/installation.html)
-   installed.
-
-3. `cd` into the repository cloned earlier:
    `cd python-zulip-api`
+   `git remote add upstream https://github.com/zulip/python-zulip-api.git`
+   `git fetch upstream master`
```

Comment:
Use a single code block, and no reason to explicitly specify `master` (especially since that may be changing soon):
````suggestion
    ```
    git clone https://github.com/<your_username>/python-zulip-api.git
    cd python-zulip-api
    git remote add upstream https://github.com/zulip/python-zulip-api.git
    git fetch upstream
    ```
````